.mailmap

```diff
@@ -565,6 +565,7 @@ Robert Habermeier <rphmeier@gmail.com>
 Robert Millar <robert.millar@cantab.net>
 Roc Yu <rocyu@protonmail.com>
 Rohit Joshi <rohitjoshi@users.noreply.github.com> Rohit Joshi <rohit.joshi@capitalone.com>
+Ross Smyth <18294397+RossSmyth@users.noreply.github.com> <crs2017@gmail.com> <rsmyth@electrocraft.com> Christopher Smyth <crs2017@gmail.com>
```

Comment:
I think you need to split these into several lines. Look, for example, at the entry for Rémy above on line 552.

Reply:
Ah I think I understand